.circleci/config.yml

```diff
@@ -0,0 +1,234 @@
+version: 2.1
+orbs:
+  git: opuscapita/git@0.0.3
+  github-release: izumin5210/github-release@0.1.1
+
+  gcp-gcr: circleci/gcp-gcr@0.13.0
+
+
+jobs: 
+   create-and-push-builder:
+     #executor: gcp-gcr/default
+     docker:
+       - image: circleci/buildpack-deps:stretch
+     steps:
+       ### This step is to enable local testing with circleci cli tool
+       - run:
+          name: Local build handling
+          command: |
+            if [[ ${CIRCLE_SHELL_ENV} =~ "localbuild" ]]; then
+              if [ -d /workdir ]; then
+                ln -s /workdir /tmp/_circleci_local_build_repo
+              else
+                echo "Run this local build using: circleci build -v \$(pwd):/workdir"
+                exit 1
+              fi
+            fi
+       - checkout
+       - gcp-gcr/gcr-auth
+       - setup_remote_docker:
+          docker_layer_caching: true
+       - gcp-gcr/build-image: 
+           extra_build_args: --target builder
+           image: taraxa-node/builder
+       - gcp-gcr/push-image:
+           image: taraxa-node/builder
+   build-mac:
+     macos:
+       xcode: 12.0.0
+     environment:
+        CLANG_VERSION: "1:10.0-50~exp1"
+        GO_VERSION: "1.13.7"
+        CMAKE_VERSION: "3.16.3-1ubuntu1"
+        GCC_VERSION: "4:9.3.0-1ubuntu2"
+        GFLAGS_VERSION: "2.2.2-1build1"
+        JSONCPP_VERSION: "1.7.4-3.1ubuntu2"
+        JSONRPCCPP_VERSION: "0.7.0-1build3"
+        SCRYPT_VERSION: "1.21-3"
+     steps: 
+        - checkout
+        - run: 
+           name: Checkout Submodules
+           command: |
+              git submodule sync
+              git submodule update --init --recursive --jobs 8
+        - run: 
+           command: brew update
+        - run: 
+           command: brew install coreutils 
+        - run:
+           command: |
+              brew install go  ccache gflags &&
+              brew install libscrypt libtool cmake libjson-rpc-cpp conan
+        - restore_cache:
+           keys:
+            - conan-cache-v2-mac--{{ checksum "conanfile.py" }}
```

Comment:
Is there a reason to have the double `--` instead of just one?
if not, I think single would be better. Also update all other entries.
if yes, just resolve this